src/app/contextBroker/CMakeLists.txt

```diff
@@ -91,7 +93,9 @@ ELSEIF(${DISTRO} STREQUAL "Fedora_20")
   TARGET_LINK_LIBRARIES(contextBroker ${STATIC_LIBS} -lmicrohttpd -lmongoclient -lboost_thread -lboost_filesystem -lboost_system -lpthread)
 ELSE()
   MESSAGE("contextBroker: Other DISTRO: '${DISTRO}'")
-  TARGET_LINK_LIBRARIES(contextBroker ${STATIC_LIBS} -lmicrohttpd -lmongoclient -lboost_thread-mt -lboost_filesystem-mt -lboost_system-mt)
+  TARGET_LINK_LIBRARIES(contextBroker ${STATIC_LIBS} -lmicrohttpd -lmongoclient -lboost_thread -lboost_filesystem -lboost_system -lpthread)
+  INSTALL(PROGRAMS ${CMAKE_SOURCE_DIR}/etc/init.d/contextBroker.ubuntu DESTINATION /etc/init.d RENAME contextBroker)
```

Comment:
Maybe installation should be done in a make target (e.g. ubuntu_install) instead that in the CMakeLists.txt as part of the build process?

(Not sure if the above make sense... I'm not an expert in CMake :))